packages/primitives/popover/src/popover.utils.ts

```diff
@@ -135,9 +133,13 @@ export function getArrowPositionParams(
         } else if (sideAndAlign.align === RdxPopoverAlign.Center) {
             posParams.top = `calc(50% - ${arrowWidthAndHeight.height / 2}px)`;
         } else if (sideAndAlign.align === RdxPopoverAlign.End) {
-            posParams.bottom = `${(triggerWidthAndHeight.height - arrowWidthAndHeight.height) / 2}px`;
+            posParams.top = `calc(100% - ${(triggerWidthAndHeight.height + arrowWidthAndHeight.height) / 2}px)`;
         }
     }
 
     return posParams;
 }
+
+export function isRdxPopoverDevMode() {
```

Comment:
will this be removed before merging into the main branch?

Reply:
yes

Reply:
removed